src/app/app.module.ts

```diff
@@ -11,14 +11,24 @@ import { ContentModule } from './modules/content/content.module';
 import { NavbarModule } from './modules/navbar/navbar.module';
 import { PagesModule } from './modules/pages/pages.module';
 import { UsersModule } from './modules/users/users.module';
-import { ReactiveFormsModule } from '@angular/forms';
+import {FormsModule, ReactiveFormsModule} from '@angular/forms';
 import { KeycloakAngularModule, KeycloakService } from 'keycloak-angular';
 import { initializeKeycloak } from './keycloak/keycloak.init';
+import { TeacherHomePageComponent } from './modules/teacher/teacher-home-page/teacher-home-page.component';
+import {MatOptionModule} from '@angular/material/core';
```

Comment:
Material design imports go into the MaterialModule https://github.com/Clean-CaDET/platform-tutor-ui-web/blob/master/src/app/infrastructure/material.module.ts